.github/workflows/scripts/unix_build.sh

```diff
@@ -4,7 +4,26 @@ python3 -m pip uninstall taichi taichi-nightly -y
 python3 -m pip install -r requirements_dev.txt
 python3 -m pip install -r requirements_test.txt
 git fetch origin master
-
+export SCCACHE_DIR=`pwd`/sccache_cache
+export SCCACHE_CACHE_SIZE="128M"
+echo sccache dir: $SCCACHE_DIR
+if [[ $OSTYPE == "linux-"* ]]; then
+    wget https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz
+    tar -xzf sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz
```

Comment:
Is musl build usable on GNU linux?

Reply:
Seems like it can, circa https://www.musl-libc.org/faq.html

Reply:
It seems in this case it's working fine.